srcpkgs/mako/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'mako'
 pkgname=mako
-version=1.4.1
+version=1.5
 revision=1
 build_style=meson
-configure_args="-Dzsh-completions=true -Dapparmor=true -Dwerror=false"
+configure_args="-Dzsh-completions=true -Dwerror=false"
 hostmakedepends="pkg-config scdoc wayland-devel wayland-protocols"
 makedepends="wayland-devel wayland-protocols pango-devel cairo-devel
- elogind-devel gdk-pixbuf-devel"
+ basu basu-devel gdk-pixbuf-devel"
+depends="basu"
```

Comment:
```suggestion
 basu-devel gdk-pixbuf-devel"
```

devel package already pulls in the normal one, and the dependency on `basu` should be detected automatically.

Reply:
Thanks for the tip